src/builder.ts

```diff
@@ -335,6 +346,13 @@ export class SchemaBuilder {
           }
           const fieldName = resolvedConfig.alias || field.name
           const type = resolvedConfig.type || field.outputType.type
+
+          if (
+            !this.assertOutputTypeIsDefined(typeName, fieldName, type, stage)
```

Comment:
```suggestion
            !this.assertOutputTypeIsDefined(fieldType, fieldName, type, stage)
```

And instances later. Think its clearer to associate that this type name is the type name of the field type.